app/assets/stylesheets/application.scss

```diff
@@ -41,7 +41,15 @@ body p {
     font-size: 20px;
     margin-top: 10px;
 }
-
+.comment-list{
+    width: 100%;
+    height: 70px;
```

Comment:
Why are you setting the height explicitly in terms of px?  

In more than 75% of the cases you should not set the height of width in px. You should just set the right font size and give padding / marin in px. That should do.

Setting the height explicitly makes responsive design very difficult. You would have tested only in a few displays resolutions and wont look right in the others.

Reply:
Yes this is wrong ! i understood , like even in small displays if there is a big comment of 5 lines maybe then it breaks ! I will remove this .